src/05-derivatives/01-introduction.md

```diff
@@ -28,6 +28,75 @@ in [Derived dataset and pipeline description][derived-dataset-description].
     to ensure that the values are still relevant and appropriate to the type of
     output data.
 
+## Model-based grouping of derivatives 
+
+-  Here, a *model* is defined as a process via which parameters that describe
+   the data are fit/derived. This could include, but is not limited to things 
+   like: 
+    
+    1. The beta weights for different kinds of events in an event-related design 
+       in fMRI. 
+    1. The six components of the diffusion tensor in DTI. 
+    1. Etc. 
+
+   Model-based derivatives SHOULD be saved in a directory named 
+   `model-<modelname>` that is placed under the datatype from which the model 
+   was derived. A metadata file, `models.tsv` is OPTIONAL, accompanied by 
+   a `models.json` file that is REQUIRED, if and only `models.tsv` is present.
+   The `models.json` is a dictionary for the `models.tsv` columns. 
+   The purpose of the RECOMMENDED `models.tsv` file is to describe properties of the models 
+   such as the datatype from which they are derived and a human-readable description of the 
+   model. An additional metadata file, `model-<modelname>.json` is REQUIRED and describes the metadata for model fitting procedures, etc. for each model. 
+   
+   For example, if the model is principally derived from DWI data. 
+   It would go into the following structure: 
+   
+    ```Text
+    <pipeline_name>/
+        sub-<label>/
+            <datatype>/
+                model-<modelname>/
+                    <model files>
+                    <model metadata files>
+                model-<modelname>_model.json
+                models.tsv
+                models.json
+    ```
+
+    The contents of the `model-<modelname>/` folders can differ widely between models 
+    and modalities and are described in the relevant modality-specific derivative 
+    specifications. 
+
+    For a concrete example, consider the following multi-modal dataset:
+
+    ```Text
+    pipelineush-v2022a/
+        models.tsv
+        models.json
+        sub-01/
+            dwi/
+                model-DTI/
+                    sub-01_model-DTI_param-tensor_model.nii.gz
```

Comment:
@arokem @oesteban one question that came up today about this is why the suffix is `_model.nii.gz` given that both the folder and the file already contain the word `model` in `func` the suffix could be `_bold.nii.gz` perhaps in dw it could be `_dwi.nii.gz`?

Reply:
As I commented above, this PR does not intend to get into that question. I have updated the template above to be agnostic to this detail.

As for this particular example, I'd be fine by removing this example, but perhaps indicating that the suffix here is fictional and does not represent any existing specification would be sufficient.

Finally, a discussion about the BEP016 proposal about this suffix would be necessary.

Reply:
Okay, it was also cheap to replace `_model` with `_<suffix>` because it is totally true this example is a bit distracting (hopefully that will be fixed by #1282).